assets/templates/contact-policy.html

```diff
@@ -0,0 +1,197 @@
+      type="image/x-icon"
+      href="https://www.fromdoppler.com/favicon.ico"
+    />
+    <link
+      rel="shortcut icon"
+      type="image/x-icon"
+      href="https://www.fromdoppler.com/favicon.ico"
+    />
+  </head>
+
+  <body data-active-menu="34" class="dp-library">
+    <div class="dp-container">
+      <div class="dp-rowflex">
+        <div class="col-sm-12 m-t-24">
+          <header class="m-b-30">
+            <a
+              href="index.html"
+              class="dp-button button-medium secondary-green button--round"
+            >
+              Back
+            </a>
+          </header>
+        </div>
+      </div>
+    </div>
+
+    <header class="hero-banner">
+      <div class="dp-container">
+        <div class="dp-rowflex">
+          <div class="col-sm-12 col-md-12 col-lg-12">
+            <nav class="dp-breadcrumb">
+              <ul>
+                <li>
+                  <a href="#">Panel de Cotrol</a>
+                </li>
+                <li>Política de Contacto</li>
+              </ul>
+            </nav>
+            <h2>Política de Contacto</h2>
+          </div>
+          <div class="col-lg-7 col-md-7 col-sm-12">
+            <p>
+              Here you'll find all your Subscription Forms including the ones
+              that are in draft, published or with unpublished changes. You
+              currently have 39 Forms. You can create up to
+              <a href="#">HELP.</a>
+            </p>
+          </div>
+        </div>
+        <span class="arrow"></span>
+      </div>
+    </header>
+    <section class="dp-container">
+      <div class="dp-rowflex">
+        <div class="col-lg-6 col-md-12 col-sm-12 m-b-24">
+          <form class="dp-contact-policy-form">
+            <fieldset>
+              <legend>Política de contacto</legend>
+              <ul class="field-group">
+                <li class="field-item">
+                  <div class="dp-switch">
+                    <input
+                      type="checkbox"
+                      id="oneCheck"
+                      checked=""
+                      type="switch"
+                    />
+                    <label for="oneCheck">
+                      <span></span>
+                    </label>
+                  </div>
+                  <label for="switch"
+                    >Limitar la cantidad maxima de envios a un mismo
+                    Suscriptor</label
+                  >
+                </li>
+                <li class="field-item">
+                  <div class="dp-item-block">
+                    <div>
+                      <span>No enviar mas de 20:</span>
+                      <input
+                        type="number"
+                        name="number"
+                        id="correos electrónicos"
+                        placeholder=""
+                      />
+                    </div>
+
+                    <div>
+                      <span>Emails en un período de</span>
```

Comment:
Just a very minor detail, the word Emails belongs to the input above. 
No enviar mas de 20: ___ Emails
en un período de ___ días